### PR TITLE
WRN-6747: Fixed tooltip arrow position and shape

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Fixed tooltip arrow position 
Before:
image
After:
image

Fixed tooltip arrow border width to be the same as the tooltip label's border
Before:
image image

After:
image image

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-6747

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### CODE REVIEW

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

File: TooltipDecorator/Tooltip.module.less [hunk 1]
```diff
@@ -6,12 +6,12 @@
...
 	background-image:
-		linear-gradient(to @edge, @moon-tooltip-border-color @moon-tooltip-border-width, transparent @moon-tooltip-border-width), // Edge line
-		linear-gradient(to @point @edge, @moon-tooltip-bg-color 50%, @moon-tooltip-border-color 50%, @moon-tooltip-border-color ~"calc(50% + 1px)", transparent ~"calc(50% + 1px)"); // Diagonal line + fill
+		linear-gradient(to @edge, @moon-tooltip-border-color @moon-tooltip-border-width/2, transparent @moon-tooltip-border-width), // Edge line
+		linear-gradient(to @point @edge, @moon-tooltip-bg-color 50%, @moon-tooltip-border-color ~"calc(50% + 0.5px)", @moon-tooltip-border-color ~"calc(50% + 1px)", transparent ~"calc(50% + 1px)"); // Diagonal line + fill
 }
 .centerTail (@edge, @point) {
 	background-image:
-		linear-gradient(to @point @edge, @moon-tooltip-bg-color 50%, @moon-tooltip-border-color 50%, @moon-tooltip-border-color ~"calc(50% + 1px)", transparent ~"calc(50% + 1px)"); // Diagonal line + fill
+		linear-gradient(to @point @edge, @moon-tooltip-bg-color 50%, @moon-tooltip-border-color ~"calc(50% + 0.5px)", @moon-tooltip-border-color ~"calc(50% + 1px)", transparent ~"calc(50% + 1px)"); // Diagonal line + fill
```
Comment (jeonghee27) [inline TooltipDecorator/Tooltip.module.less:14]: Do we have to change this part or what if we don't?

Comment (daniel-stoian-lgp) [reply TooltipDecorator/Tooltip.module.less:14]: Hello Jeonghee, here is what it looked like before. The top corners of the arrow were a little bit too deep in the tooltip body + the border width is thicker than the tooltip body's border.
Without modification
And now:
With modification
If you think this change is not necessary, let us know

Comment (jeonghee27) [reply TooltipDecorator/Tooltip.module.less:14]: I understand. Thanks for the kind explanation.

**Review by jeonghee27 — APPROVED** [review]
LGTM

**Pull request merged**